test/prefix-url.js

```diff
@@ -33,3 +41,25 @@ test('prefixUrl option', async t => {
 
 	await server.close();
 });
+
+test('prefixUrl option in browser', withPage, async (t, page) => {
```

Comment:
I was just trying to keep the tests that were already in Cypress and provide a starting point for testing in the browser. I don't see the point in testing prefixUrl in node at all since how it works is affected by `document`, but I'm also not willing to refactor all of the tests right now.

Reply:
> I was just trying to keep the tests that were already in Cypress and provide a starting point for testing in the browser.

Right, I'm just asking to keep the browser tests in a separate file, as the Cypress tests were.

> I don't see the point in testing prefixUrl in node at all since how it works is affected by document

It's very important to test `prefixUrl` in Node, precisely because its behavior is affected by the presence of `document`. We support `prefixUrl` in Node even you do not define a `document`.

Reply:
I suppose I didn't word that very well. It is true that Node is not a target environment. But people are using Ky in Node anyway. They have complained when we've broken Node support in the past and thus far we have fixed it since it was easy. More importantly, we support Web Workers, which is a browser-based environment without a `dicument`. We also just added support for [Deno](https://github.com/denoland/deno}.

So, what I should have said is, it's very important to test Ky in a non-DOM environment. Doing that in Node happens to be a convenient way to do so that also aligns with things our users are doing. I don't see a good reason to remove the Node tests, but I guess it would be fine if we add some Web Worker tests, etc.

Reply:
Ok, gotcha. I will move these to a `browser.test.js` or something like that.